lib/saxerator/configuration.rb

```diff
@@ -69,8 +78,14 @@ def put_attributes_in_hash?
 
     def raise_error_if_using_put_attributes_in_hash_with_xml
       if @output_type != :hash && @put_attributes_in_hash
-        raise ArgumentError.new("put_attributes_in_hash! is only valid when using output_type = :hash (the default)'")
+        raise_exeption("put_attributes_in_hash! is only valid when using output_type = :hash (the default)'")
       end
     end
+
+    private
+
+    def raise_exeption(text)
```

Comment:
This is misspelled `raise_exception` but also, I don't think this extraction is necessary. `raise ArgumentError, ""` is not much duplication (or we'd have these sorts of methods all over the place)